### PR TITLE
Dashboard sidebar CSS and markup updates

### LINKED ISSUE #5341: Dashboard sidebar
### Descriptive summary

Sidebar UI needs some cleanup.

### Expected behavior

image

### Actual behavior

image

### PR DESCRIPTION
**Pull request by adamjarling**
Fixes #5341 

This updates the Dashboard markup and sidebar CSS styles for better UI display.

image

Changes proposed in this pull request:
* Updated Bootstrap 4 implementation

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: app/assets/stylesheets/hyrax/_header.scss [hunk 1]
```diff
@@ -2,8 +2,9 @@
   margin-top: $search-bar-margin-top;
 }
 
-#masthead {
-  z-index: 1;
+header.navbar-header,
```
[the remark this answers is not in the document]
Comment (adamjarling) [reply app/assets/stylesheets/hyrax/_header.scss]: Changed the classname in a commit `amend` to `top-header`.

File: app/assets/stylesheets/hyrax/dashboard.scss [hunk 2]
```diff
@@ -113,14 +113,14 @@ body.dashboard {
...
 
-
     .btn {
       margin-bottom: $input-btn-padding-y;
     }
   }
 
   // the span selector is required because Blacklight is using it.
-  .list-filters-label, span.constraints-label {
+  .list-filters-label,
```
[the remark this answers is not in the document]
Comment (adamjarling) [reply app/assets/stylesheets/hyrax/dashboard.scss:122]: This has to do with presenters and other things I don't understand, so not going to add new classnames for fear of breaking something.

File: app/assets/stylesheets/hyrax/sidebar.scss [hunk 3]
```diff
@@ -18,16 +18,19 @@ $gutter-width: $grid-gutter-width/2;
 // .sidebar is a class on the div
 .sidebar {
   background-color: $admin-sidebar-background-color;
-  padding-top: $admin-vertical-padding;
+  padding-bottom: 2rem;
```
[the remark this answers is not in the document]
Comment (adamjarling) [reply app/assets/stylesheets/hyrax/sidebar.scss:22]: Updated in a commit `amend`.

**Comment by Dananji** [comment]
@adamjarling The code changes looks good to me :+1: 
And the sidebar looks a lot better now :tada: 

One thing I noticed, is that when clicked on a link inside the collapsible sections the collapsed section on the sidebar won't say open. 

In Nurax-dev when clicked on `Profile`, -> `Your Activity` section stays open and highlights `Profile` link all the way through;
sidebar-nurax

Locally it looks as follows;
sidebar-bl7

And also the entire sidebar is not collapsible from the arrow icon on the top-right corner of the sidebar.

**Comment by adamjarling** [comment]
Hey @Dananji Thanks for the feedback.  So the issues you are referring to:

- The sidebar opening / closing
- Sidebar navigation remaining open or identified by an 'active' state

are both not working in the `bl7bootstrap4` branch either.  I tried to initially trouble-shoot the main sidebar open / close issue by tracking down the jQuery event listener, and looked like the event wasn't getting picked up by the JS file where the listener callback function was set up.  So... I believe there are deeper JS issues which honestly I avoided in this ticket, just to move on something visual.  

How about we scope this issue to just the CSS?   I'll reflect that in the PR title and description.

**Review by Dananji — APPROVED** [review]
@adamjarling Yes, those were the issues I was referring to. And yes, that sounds like a good plan.  And for CSS updates this looks great :100: 

Maybe we create a new ticket for JS updates?

**Pull request merged**